src/components/Terminal.tsx

```diff
@@ -46,7 +46,9 @@ export default function Terminal(props: any) {
     errorMessage={errorMessage}
     enableInput={enableInput}
     showControlBar={showControlBar}
-    defaultHandler={defaultHandler}/>
+    defaultHandler={defaultHandler}
+    editorStyle={props.editorStyle}
```

Comment:
Can you document what this prop is used for? If it's meant to supply custom css to the editor, can it be done through custom themes?

Reply:
https://github.com/bony2023/react-terminal/pull/255#issuecomment-1371515722